### PR TITLE
Replace user guide links

### PR DESCRIPTION
**Pull request by droumis**
This is in progress, there are many more to fix. :(

### CODE REVIEW

**Comment by droumis** [comment]
@philippjfr, for updates to accessing/manipulating the url (how_to/state/url), I put the port number after the domain `.org`. Is that incorrect? Before it was after the file path `.html`

File: doc/how_to/state/url.md [hunk 1]
```diff
@@ -12,16 +12,16 @@ When starting a server session Panel will attach a `Location` component which ca
...
-* **``hash_``** (string): hash part of the url e.g. '#interact'.
+* **``hash_``** (string): hash part of the url e.g. '#margin-parameter'.
 * **``reload``** (bool): Whether or not to reload the page when the url is updated.
     - For independent apps this should be set to True.
     - For integrated or single page apps this should be set to False.
 
 ### Readonly
 
-* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org/user_guide/Interact.html:80?color=blue#interact'.
+* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org:443/how_to/layout/spacing.html?color=blue#margin-parameter'.
```
Comment (philippjfr) [inline doc/how_to/state/url.md:24]: Yes, port after hostname, although in this case I'd just leave it out

```suggestion
* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org/how_to/layout/spacing.html?color=blue#margin-parameter'.
```

**Comment by philippjfr** [comment]
Note I made a lot of fixes here: https://github.com/holoviz/panel/pull/4872

File: doc/how_to/state/url.md (hunk 1 above)
Comment (droumis) [reply doc/how_to/state/url.md:24]: right, in the default port case, one would typically leave it out, but we want to demonstrate specifying/accessing the port, so I think leave it in here

**Review by philippjfr — APPROVED** [review]
Thanks!

**Pull request merged**